storybookjs/storybook
# start haul/react-native using named binaries instead of cli.js

yarn workspaces breaks storybook due to absolute path of cli.js changes to workspace root.

Issue:

yarn workspaces causes node modules to end up in the workspace root. And in general node resolves node_modules by searching in parent folders. Because of this we cannot assume the location of cli.js to be known.

It's important to note that this issue is not isolated to workspaces, but rather related to the naive view of how node_modules are resolved that causes the bug.

## What I did

Change to use react-native or haul using installed binary name. These binaries are callable even when not installed globally when storybook is called through npm scripts. 

## How to test

Put a react native app w storybook as a package in a yarn workspace repo. Try to run yarn storybook. 

Is this testable with jest or storyshots?

no

Does this need a new example in the kitchen sink apps?

no

Does this need an update to the documentation?

no

If your answer is yes to any of these, please make sure to include it in your PR.
